new_game.py

```diff
@@ -0,0 +1,139 @@
+# Python 2
+from PyDictionary import PyDictionary
+import random, time, os
+
+class Data(object):
+
+    # Builds the word database
+    def collect_words(word_file="gre_word_file.txt"):
+        wordlist = open(word_file).read().splitlines()
+        return wordlist
+
+    def __init__(self, words=collect_words()):
```

Comment:
So this would also be good if the user decides to replay, he won't deal with repeated words from his previous game right?